pyanaconda/payload/__init__.py

```diff
@@ -81,6 +81,23 @@ def __init__(self, data):
     def first_payload_reset(self):
         return self._first_payload_reset
 
+    @property
+    def is_ssl_verification_enabled(self):
```

Comment:
I would say it makes a standard because thanks to the fact that inst.nosslverify is set to the ``ksdata.method`` and then it doesn't make sense to use the flag with the method later.
However, you are right that this will be solved better later. So it is removed.